builder/proxmox/common/config.go

```diff
@@ -64,6 +64,7 @@ type Config struct {
 
 	TemplateName        string `mapstructure:"template_name"`
 	TemplateDescription string `mapstructure:"template_description"`
+	ConvertToTemplate   bool   `mapstructure:"convert_to_template"`
```

Comment:
The default of this configuration will always be false which will indeed change the default behavior of the builder. Also, in the `prepare` method you're explicitly setting it to false as default. 

If the intention is to not change the default behavior, the default should be `true`. In this case, you can: 
- Use [Trilian](https://github.com/hashicorp/packer-plugin-sdk/blob/main/template/config/custom_types.go#L8), a custom type available in the SDK for this type of configuration
- Rename it to `skip_convert_to_template` that will skip converting to template when set to true. The default will be automatically false.

Reply:
@sylviamoss Thanks for the feedback, this should be resolved now.